tsdb/engine/tsm1/wal_test.go

```diff
@@ -445,43 +371,3 @@ func BenchmarkWALSegmentWriter(b *testing.B) {
 		}
 	}
 }
-
-func BenchmarkWALSegmentReader(b *testing.B) {
```

Comment:
ah yes, got a little too aggressive. I thought it was for the old `WALSegmentIterator` which is longer there. Will add back in.